CMakeLists.txt

```diff
@@ -75,13 +72,12 @@ set(CMAKE_AUTORCC ON)
 # Enable maximum compression
 set(AUTORCC_OPTIONS -compress 9 -threshold 0)
 
-
 if(NOT Qt5Widgets_VERSION VERSION_LESS "5.9")
```

Comment:
not less, should be `VERSION_GREATER_EQUAL`

Also we might work around this in another way, or not care at all.

Reply:
Ok, I've removed the condition.